.travis.yml

```diff
@@ -31,6 +31,8 @@ before_script:
 
 script:
     - grunt travis
+    # Fail if dist has not been compiled
```

Comment:
Failed on Travis, so hopefully 5713e34 and 32df3a5 will make it more clear why it failed